src/SessionSetup/ConnectToStadiaWidget.cpp

```diff
@@ -92,25 +101,69 @@ ConnectToStadiaWidget::ConnectToStadiaWidget(QWidget* parent)
                    &ConnectToStadiaWidget::UpdateRememberInstance);
   QObject::connect(ui_->refreshButton, &QPushButton::clicked, this,
                    [this]() { emit InstanceReloadRequested(); });
+  QObject::connect(ui_->instancesFilterLineEdit, &QLineEdit::textChanged, &instance_proxy_model_,
+                   &QSortFilterProxyModel::setFilterFixedString);
+  QObject::connect(ui_->refreshButton_2, &QPushButton::clicked, this,
+                   [this]() { emit InstanceReloadRequested(); });
+  QObject::connect(ui_->comboBox, QOverload<int>::of(&QComboBox::activated), this,
+                   &ConnectToStadiaWidget::ProjectComboBoxActivated);
+  QObject::connect(ui_->allInstancesCheckBox, &QCheckBox::stateChanged, this, [this]() {
+    QSettings settings;
+    settings.setValue(kAllInstancesKey, ui_->allInstancesCheckBox->isChecked());
+  });
 
   SetupStateMachine();
 
   // The instance settings (mainly project selection and "all instances") is currently hidden behind
   // the "--enable_project_selection" flag.
   // TODO(b/190670843): Clean this up when removing the "--enable_project_selection" flag.
   if (absl::GetFlag(FLAGS_enable_project_selection)) {
-    // TODO(b/199717843): The instanceFilterLineEdit is currently hidden because it is lower
-    // priority as the rest of the UI.
-    ui_->instancesFilterLineEdit->hide();
     // While the "--enable_project_selection" is used, there are 2 refresh buttons in the .ui file.
     // "refreshButton" is used for the old ui, "refreshButton_2" is used for the new ui.
     ui_->refreshButton->hide();
+
+    ui_->comboBox->addItem("Default Project", QVariant());
+    QVariant saved_project_id = settings.value(kSelectedProjectIdKey);
+    if (saved_project_id.isValid()) {
```

Comment:
This is done so the last saved project is displayed instantly before the project list is loaded, right? This should be stated in a comment.

Reply:
done